internal/resolver/delegatingresolver/delegatingresolver.go

```diff
@@ -36,17 +36,14 @@ import (
 
 var (
 	// The following variable will be overwritten in the tests.
-	httpProxyFromEnvironment = http.ProxyFromEnvironment
+	HttpsProxyFromEnvironment = http.ProxyFromEnvironment
```

Comment:
is this need to be exported?

Reply:
Yes, It needs to be exported for the e2e style tests.